crowbar_framework/config/application.rb

```diff
@@ -19,8 +19,7 @@
 
 module Crowbar
   class Application < Rails::Application
-    # Explicitely eager load /lib/crowbar/lock so we can use SharedNonBlocking
-    # with threading without hitting circular dependencies
+
     config.eager_load_paths += Dir["#{config.root}/lib/crowbar/lock"]
```

Comment:
did you want to remove the actual eager loading as well instead of just the comment that explains why the line is there?